kubernetes/linux/setup.sh

```diff
@@ -57,7 +57,7 @@ sudo tdnf install jq-1.6-1.cm2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo tdnf install libcap -y
 
-sudo tdnf install telegraf-1.25.2 -y
+sudo tdnf install telegraf-1.26.0 -y
```

Comment:
did we test this upgrade works as expected?

Reply:
Yes have tested this...Working as before. Any other scenarios we would like to test?

Reply:
thanks, lets also look the perf usage and verify no perf regressions, if you havent verified already.

Reply:
I did but I added the ruby changes as well now so once that image is ready I will attach the screenshots

Reply:
Memory - workingset
![image](https://github.com/microsoft/Docker-Provider/assets/18661789/fd56649e-2f83-48c2-b977-95e89e1309c1)

Cpu usage
![image](https://github.com/microsoft/Docker-Provider/assets/18661789/68657577-7a22-48ed-9584-b6ae564eb32e)

Reply:
@jatakiajanvi12 not sure what these graphs mean, can you add legends ?

Reply:
@pfrcks Updated the comment....Do you want me to add details in the graph?